Source/CurrencyCloud/Client.cs

```diff
@@ -214,10 +194,15 @@ await retryPolicy.ExecuteAsync(
             {
                 try
                 {
-                    if (attempts > 0)
+                    if (attempts == 1)
                     {
                         await AuthorizeAsync();
                     }
+                    else if (attempts > 1)
+                    {
+                        await AuthorizeAsync(true);
```

Comment:
named parameter required

Reply:
there are not format rules for this project, but anyway I thought this rules only applies to cases when passing null?